api/config/service/temporal/v1/temporal.proto

```diff
@@ -17,10 +17,4 @@ message Config {
 
 message ConnectionOptions {
   bool use_system_ca_bundle = 1;
-
-  // If set to true, the Temporal client will attempt to connect to the Temporal server when GetNamespaceClient is
-  // called. In general the health check should not be enabled unless the only purpose of the configured Clutch instance
-  // is to execute Temporal worker code, as it may block Clutch from running at all in the event Temporal is down. This
-  // can create a circular dependency if Clutch also hosts tools that could be used to mitigate a Temporal outage.
-  bool enable_health_check = 2;
```

Comment:
Do we still want an option to disable lazy connect? In the event we rather panic / shutdown on startup? I suppose there are many different ways we could observe failure to connect, panic metrics, other metrics that we can alarm on such as workflow start latencies and etc.

Reply:
You can explicitly call GetConnection and save the connection in that case.